cpp/syslog.cc

```diff
@@ -0,0 +1,123 @@
+/** \brief A tool for logging to syslogd in a shell script.
```

Comment:
Could there be conflicts with any other program named syslog in PATH?

Reply:
No, if at all the demon is called syslogd and, I checked, there is no system tool called syslog.